src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.FunctionalTest/tests/runTest.ts

```diff
@@ -0,0 +1,30 @@
+// /* --------------------------------------------------------------------------------------------
+//  * Copyright (c) Microsoft Corporation. All rights reserved.
+//  * Licensed under the MIT License. See License.txt in the project root for license information.
+//  * ------------------------------------------------------------------------------------------ */
+
+// This file is used at the command line to download VSCode insiders and run all of our functional tests.
+
+import * as path from 'path';
+import { runTests } from 'vscode-test';
+
+async function main() {
+    try {
+        const extensionDevelopmentPath = path.resolve(__dirname, '../../../src/Microsoft.AspNetCore.Razor.VSCode.Extension/');
+        const extensionTestsPath = path.resolve(__dirname, './index');
+        const testAppFolder = path.resolve(__dirname, '../../testapps');
+
+        // Download VS Code, unzip it and run the integration test
+        await runTests({
+            extensionDevelopmentPath,
+            extensionTestsPath,
+            version: 'insiders',
```

Comment:
There's a bug in latest VSCode that impacts functional test reliability. It's fixed in the insiders build so we're requiring that for now.